File: java/2543.java
import java.io.IOException;
import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) throws IOException {
        int n, id, total, l, m;
        Scanner input = new Scanner(System.in);
        
        while (input.hasNext()) {
            n = input.nextInt();
            id = input.nextInt();
            total = 0;
            
            for (int i = 0; i < n; i++) {
                l = input.nextInt();
                m = input.nextInt();
                
                if (l == id && m == 0) {
                    total++;
                }
            }
            
            System.out.println(total);
        }
        
        input.close();
    }
}

File: src/py/2543.py
try:
    while True:
        total = 0
        n, gameplayId = map(int, input().split())
        
        for _ in range(n):
            l, m = map(int, input().split())
            
            if l == gameplayId and m == 0:
                total += 1
        print(total)
except EOFError:
    pass
